index.html

```diff
@@ -6,15 +6,59 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    
+
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
-</head>
+    </head>
 
-<body>
+<body class="sm-col-12 ">
 
     <!-- Add your markup here -->
+    <nav class="nav-content sm-col-12 no-padding" >
+        <div class=" no-padding col-6">
+            <div class="left-side ">
+                <h5 class="font login-font">PRICKLES & CO.</h5>            
+            </div>
+            <ul class="menu hide ">
+                <input type="checkbox" class="toggler"></input>
+                <li><a>Home</a></li>
+                <li><a>Shop</a></li>
+                <li><a>About</a></li>
+                <li><a>FAQ</a></li>
+                <li><a>Contact</a></li>
+            </ul>
+        </div>
+        <div class="sm-col-6 no-padding right-side">
+            <h2 class="hide"> Call Us Now 123-456-7890</h2>
+            <div class="right-side sm-col-6">
+                <img src="../img/user-logo.png" class="sm-col-1">
+                <a href="https/www.google.com" class="login-font">LogIn</a>
+            </div>
+        </div>
+    </nav>
+    <section class="first-section sm-col-12">
```

Comment:
This is now the start of your main part of your website - wrap this in the semantic `<main>` html element to indicate that this is the case.